eng/Version.Details.xml

```diff
@@ -11,302 +11,302 @@
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Security.Cryptography.Pkcs" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Security.Cryptography.Xml" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Security.Permissions" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Security.Principal.Windows" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.ServiceProcess.ServiceController" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Text.Encodings.Web" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Text.Json" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Threading.Channels" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Windows.Extensions" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NETCore.App.Ref" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
```

Comment:
@ericstj and @mmitche this looks suspect. Why is there a new build of the Microsoft.NETCore.App.Ref package after what we shipped in 5.0.0❔ I'm wondering if we need to just ignore this PR until we get a real 5.0.1 build from dotnet/runtime.

Reply:
Because we staged a fix for 5.0.0 that we didn't ship. These need to be reverted to the 5.0.0 GA shas.

Reply:
Can we just wait for the bot to update this PR w/ 5.0.1 versions❔ I did the revert thing in #27697 but that wasn't downgrading from the same build as this is attempting to upgrade to. Suspect dotnet/runtime built 5.0.0 _again_ prior to 5.0.1 branding, perhaps because the 5.0.1 official builds of that repo are failing.

Reply:
Ahh yeah that was my build right before the branding update went in. I will make a pass over the entire stack and fixup these tomorrow. You can check this in for now if you'd like.